### ISSUE TITLE
IA review from Debt Resolution Team, MCP (HTML Statements)

### ISSUE THREAD

**Issue by rebeccawalsh** [department-of-veterans-affairs/va.gov-team#32652]
## What does your team need support for? Check all that apply.
- [ ] Launching one or more new unauthenticated (static) VA.gov pages
- [ ] Making revisions to one or more existing unauthenticated (static) VA.gov page
- [ ] Launching a new online form, app, or tool on VA.gov
- [x] Making revisions to an existing online form, app, or tool on VA.gov
- [ ] Something else (please add details in the next question)

## What do you need help with?
  - [ ] Content support
  - [x] IA support
  
*Please describe your need and provide any additional documents and/or links that will help us help you. If we need to add a react widget to a page, be sure to provide the code here.*

We're working on creating HTML / accessible statements for MCP, and we decided that we would offer the accessible page as the default when looking at a statement, then provide the PDF download on that page.

Our guess at the structure:
![image](https://user-images.githubusercontent.com/6361930/140964791-b01e3756-37b2-4896-a7b1-db3736bd999d.png)

A little bit more about the pages:
![image](https://user-images.githubusercontent.com/6361930/140964881-7ab3e526-d5a8-413b-980b-cb75071d7ee3.png)
  
## Supporting Artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to any epics/issues that may be helpful:

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date: Eng work slated to start next sprint if possible
- Estimated staging review date:
- Content and IA work needed by: 

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner:  Matt Self
-	Product manager: Denise Coveyduc
-	Designer: Riley Orr, Rebecca Walsh
-	FE engineer: Craig Wheeler, Justin Linn
-	Product/team Slack channel: vsa-debt

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#va-sitewide-content](https://dsva.slack.com/channels/va-sitewide-content) Slack channel and tag @RLHecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by mnorthuis** [comment]
@rebeccawalsh can you post the link to the most updated prototype of the copay experience so I can see this in context to the overall tool?  I think this is pretty straight forward and the way you have it, I just need to refresh my brain and see the full picture.

**Comment by rebeccawalsh** [comment]
Of course! I don't think we had the hifi stuff in UXPin when I made this, but it's now in UXPin [here](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/142712285?mode=i). I've added a couple of hotspots between the pages as well.

**Comment by rebeccawalsh** [comment]
@mnorthuis whoops - forgot to tag you above ^

**Comment by mnorthuis** [comment]
Hi @rebeccawalsh!  This structure you have above looks good to me, however the UXPin doesn't quite match it.   The prototype breadcrumb shows the statement being a child of the copay summary page, rather than the facility detail pages. 

To match the structure above, and to follow the standard of the breadcrumb matching the H1, the breadcrumb would actually look something like this:
- Home > Manage your VA copay bills > Your current copay balances > Your copay bill for James A. Haley Veterans' Hospital > Your June 3 statement for James A. Haley Veterans' Hospital

Your H1s have evolved a bit, and they are a bit lengthy given the facility name is in them, which will ultimately create a very lengthy breadcrumb that will wrap to a second line (ew).  So a couple of suggestions:

1. I did speak with Danielle, and you can eliminate the word "Your" from the beginning of each H1.  It's already inferred given what you have logged into, and it gets repetitive. 

2.  Beyond that, we could look at removing the facility name from the H1 on the statement page and place it below the H1 with the dates in the page content.  This would need the nod of approval from accessibility as well to make sure that removing this from the H1 doesn't create any issues. The breadcrumb would then look like this:<br> 
Home > Manage your VA copay bills > Current copay balances > Copay bill for James A. Haley Veterans' Hospital > June 3 Statement

Let me know if you have any questions/thoughts.

**Comment by rebeccawalsh** [comment]
@AngelaFowler82 @joshkimux 

Hey y'all! If you have a minute, we'd like to get your input on this issue. Breadcrumbs in the copay tool are getting a little unfortunate / long. 

What do you think about the second suggestion that Mikki made above?

**So we'd go from:**
Home > Manage your VA copay bills > Your current copay balances > Your copay bill for James A. Haley Veterans' Hospital > Your June 3 statement for James A. Haley Veterans' Hospital

**To the shortened version:**
Home > Manage your VA copay bills > Current copay balances > Copay bill for James A. Haley Veterans' Hospital > June 3 Statement
Changes: _(Remove 'your' and drop the location from the final H1, and put it in a subtitle instead)_

Is there any reason we shouldn't do this?

**Comment by joshkimux** [comment]
@rebeccawalsh That should do the trick, as it still contains key information from the `title` of the page (June 3 Statement)

**Comment by rileyorr** [comment]
Thanks for the feedback y'all! @mnorthuis I updated [our prototype's](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/142712366/simulate/sitemap?mode=i) H1's and breadcrumbs to match your recommendations. 

Danielle shared a recent draft of this tool's updated content page and it looks like the H1 is now, 'Review and pay your VA medical copay bills' - I'll update our prototype again once that page's H1 is finalized. Also, let me know if you'd like me to update [our tool's URLs and breadcrumbs documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/health-copays.md#ia-structure-urls-and-breadcrumbs-)! I didn't want to change anything in case you follow a specific format

**Comment by mnorthuis** [comment]
@rileyorr I can update the IA documentation - I think the main change is the addition of the statements page, but will go through it to make sure it's accurate.

**Comment by mnorthuis** [comment]
@rileyorr  IA documentation is updated to reflect the addition of the html statement page in the hiearchy and flow, and the breadcrumbs should now all just reflect the H1 of the page.

The only open item in this work is the financial hub page, which will hopefully be resolved tomorrow.
